packages/client/test/unit/deploy.test.ts

```diff
@@ -34,7 +34,7 @@ describe('DataUnion deploy', () => {
                 }]
             },
         }
-    })
+    }, 30000)
```

Comment:
Instead of setting timeouts for each test is it possible to set a global timeout like in mocha:
```
-s, --slow                 Specify "slow" test threshold (in milliseconds)
                                                          [string] [default: 75]
  -t, --timeout, --timeouts  Specify test timeout threshold (in milliseconds)
                                                        [string] [default: 2000]
```

Reply:
I suppose that would work too